NLibrary/Classes/ProfileMVVM/Mocks/ProfileMockRepo.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  UserProtocol.swift
+//  taylorswift
+//
+//  Created by Nathan Ngobale on 2020/03/20.
+//  Copyright © 2020 Nathan Ngobale. All rights reserved.
+//
+
+import Foundation
+import AVKit
+
+public class ProfileMockRepo: ProfileRepoProtocol {
+    var closurePathway: APIRequestResult = .succesfullRequest
+    public required init () {
+    }
+
+    public func getUserProfile(completion: @escaping(Result<ProfileModel, APIFailureResult>) -> Void) {
+        switch closurePathway {
+        case .failedRequest(_):
+            completion(.failure(.failedRequest(message: "failed getUserProfile")))
+        case .succesfullRequest:
+            let dictionary: NSDictionary = [
+                "gender": Gender.male.stringValue,
+                "name": "name",
+                "email": "email",
+                "Playlists": ["1":"x", "2": "x"]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Colons should be next to the identifier when specifying a type and next to the key in dictionary literals.](https://app.codacy.com/manual/NathanDVT/NLibrary/pullRequest?prid=5389739)